frontend/src/dashboard/dashboard-route.tsx

```diff
@@ -18,7 +27,19 @@ const DashboardRoute = () => {
         <div>Jestem zalogowany</div>
       ) : (
         <div>
-          <Header>Nie jestem zalogowany</Header>
+          <TopicArea><h1>WELCOME TO TOURNAMENT4YOU</h1></TopicArea>
```

Comment:
Is there a reason why `<h1>` tag is wrapped in `<TopicArea>` there are no styles in it

Reply:
There was no reason, so I decided to make some changes. Now it looks better in code and has more sense.